content/03.results.md

```diff
@@ -114,69 +129,74 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
+**TODO: A lot of different denominators here with no clear meaning. What is 666 vs 872 vs 54?**
+
+The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). **TODO: Clarify more what "top 50" means. Most mutations? Most commonly mutated across samples? etc.**
+_TP53_ was the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4). 
+Other canonical cancer driver genes that were frequently mutated included _BRAF_, _H3F3A_, _CTNNB1_, _NF1_, _ATRX_, _FGFR1_, and _PIK3CA_. 
+Although LGG and embryonal tumors made up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stemmed from HGGs (N = 847/872). **TODO: Is this denominator the total number of significant? Unclear what these numbers are**
+At the broad histology level, _CTNNB1_ significantly co-occurred with _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3) and _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3) in embryonal tumors, _FGFR1_ and _PIK3CA_ significantly co-occur in LGGs (OR = 76.1, 95% CI: 9.85 - 588.1, q = 3.26e-3), consistent with previous findings [@doi:10.1200/JCO.2010.31.1670; 10.1186/s40478-020-01027-z].
+We observed that 52/54 (96.3%) of tumors only have a mutation in either gene (OR = 0.188, 95% CI: 0.04 - 0.94, p = 0.0413, q = 0.0587). 
+This trend recapitulates previous observations that _TP53_ and _PPM1D_ mutations tend to be mutually exclusive in HGGs [@doi:10.1038/ng.2938].
+
+We summarized broad CNV and SV and observed that HGGs and DMGs, followed by medulloblastomas, had the most unstable genomes (**Figure {@fig:S3}A**).
+By contrast, craniopharyngiomas and schwannomas generally lacked somatic CNV.
+Together, these CNV patterns largely aligned with our estimates of tumor mutational burden (**Figure {@fig:S2}H**). 
+The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increased as breakpoint density increased (**Figure {@fig:S3}B-C**).
+We identified chromothripsis events in 41% (N = 19/46) of non-midline high-grade gliomas and in 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}D**). 
+We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors. **TODO: 15% of EACH of these groups, or TOTAL 15%?**
```

Comment:
I see these numbers are also from 3D. I'm going to rearrange this text so that it matches the order and terminology used in 3D.